docs/input/docs/reference/configuration.md

```diff
@@ -273,12 +271,16 @@ merge-message-formats:
 
 The regular expression should contain the following capture groups:
 
-*   SourceBranch - Identifies the source branch of the merge
-*   TargetBranch - Identifies the target of the merge
-*   PullRequestNumber - Captures the pull-request number
++ SourceBranch - Identifies the source branch of the merge
++ TargetBranch - Identifies the target of the merge
++ PullRequestNumber - Captures the pull-request number
```

Comment:
Why the change from `*` to `+` here?

```suggestion
+ `SourceBranch` - Identifies the source branch of the merge
+ `TargetBranch` - Identifies the target of the merge
+ `PullRequestNumber` - Captures the pull-request number
```

Reply:
This wasn't an intended change; it was because I started my changes before commit  7809b809da2268e1d11ea55bf710d727674ef2d9.  I will revert and apply your suggestion.